ec2/spot-request/main.tf

```diff
@@ -57,6 +57,7 @@ resource "aws_security_group_rule" "egress" {
   from_port         = 0
   to_port           = 0
   cidr_blocks       = ["0.0.0.0/0"]
+  ipv6_cidr_blocks  = ["::/0"]
```

Comment:
Has this been verified to work? It does not have to be an entirely separate security group rule?